MAINTAINERS

```diff
@@ -101,20 +98,16 @@ F:	ldelf/ftrace.c
 F:	lib/libutils/ext/ftrace/
 
 HiSilicon D02
-R:	Linaro <op-tee@linaro.org> [@OP-TEE/linaro]
-S:	Maintained
+S:	Orphan
 F:	core/arch/arm/plat-d02/
 
 HiSilicon HiKey (Kirin 620), HiKey960 (Kirin 960)
-R:	Linaro <op-tee@linaro.org> [@OP-TEE/linaro]
-R:	[@OP-TEE/plat-hikey]
+R:	Jerome Forissier <jerome.forissier@linaro.org> [@jforissier]
 S:	Maintained
 F:	core/arch/arm/plat-hikey/
 
 HiSilicon Poplar (Hi3798C V200)
-R:	Linaro <op-tee@linaro.org> [@OP-TEE/linaro]
```

Comment:
Is @igoropaniuk sole at HiSilicon Poplar maintenance?

Reply:
Igor has been taking care of that even after he left Linaro/GlobalLogic.